hail_search/deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hailgenetics/hail:0.2.126
+FROM hailgenetics/hail:0.2.128
```

Comment:
will I need to create a requirements file and include `google-cloud-storage` as a dependency?

Reply:
I hope not? This docker image should never be interacting with cloud storage